pkgs/by-name/vi/vidmerger/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, ffmpeg
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "vidmerger";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "TGotwig";
+    repo = "vidmerger";
+    rev = version;
+    hash = "sha256-E3Y1UaYXl6NdCMM7IepqFzWNuHaMGLCN5BvQ/lxjFoc=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    cp ${./Cargo.lock} Cargo.lock
+  '';
+
+  # Running cargo test -- . fails because it expects to have two mp4 files so that it can test the video merging functionalities
+  doCheck = false;
+
+  buildInputs = [
+    ffmpeg
+  ];
+
+  meta = with lib; {
+    description = "Merge video & audio files via CLI ";
+    homepage = "https://github.com/TGotwig/vidmerger";
+    license = licenses.unfree;
```

Comment:
The README claims it's MIT-licensed, this should be verified:
https://github.com/tgotwig/vidmerger?tab=readme-ov-file#-licensing

Reply:
yes the licensing as mentioned in the README.md is MIT but with the absence of a LICENSE.md file in the content of the repo I assumed it would be safer to mark it as unfree. I don't know if this case occurred before while packaging other software in nixpkgs. I would change the `meta.license` attribute to `licenses.mit` as your review suggests for now. Thx

Reply:
Marking it unfree is fine if that's really what it's changed to, I'd just rather hear a confirmation from upstream directly, if possible

Reply:
@eclairevoyant I already changed the `license` to `mit`. I searched where this package is already available on other package managers and registries and I found it's marked as MIT in `crates.io` and in the `aur` it' s also marked as MIT (As shown in the provided in the screenshot bellow).

![image](https://github.com/NixOS/nixpkgs/assets/88513682/2d147ea3-b313-470f-9874-51fa1a2b08df)

Reply:
In that case we should include the specific license here (Commons Clause is itself an unfree licensing clause). I'd add the Commons Clause to licenses.nix as it could be used by other projects.

Reply:
Great . thx